corehq/apps/app_manager/xform_builder.py

```diff
@@ -228,25 +227,30 @@ def _append_to_data(self, name, groups=None):
 
     def _append_to_translation(self, name, label, group=None, choices=None, label_safe=False, **params):
 
-        def get_text_node(name_, label_, group_=None, choice_name_=None, label_safe_=False, is_hint=False):
```

Comment:
[?w=1](https://github.com/dimagi/commcare-hq/pull/25653/commits/3b6760b71818a99faa990a3edd1b3b7bfce14e2e?w=1) makes this commit a bit easier to read